recipes/qjson/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "qjson" %}
+{% set version = "0.8.1" %}
+{% set sha256 = "920c94166cb91b1cf11c7d2745bdbcc8c0ea82411ca7b3732ce0b00ee2d56e98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/flavio/{{ name }}/archive/{{ version }}.tar.gz 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - qt >=4.8
+    - cmake
+  run:
+    - qt >=4.8
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libqjson.so  # [linux]
+    - test -f ${PREFIX}/lib/libqjson.dylib  # [osx]
+
+about:
+  home: http://qjson.sourceforge.net/ 
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: LGPLv2.1 
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: README.license 
+  summary: 'QJson is a qt-based library that maps JSON data to QVariant objects'
+
+  # The remaining entries in this section are optional, but recommended
```

Comment:
@ceholden can you clean the boilerplate comments from the example before I merge this?